app/main.py

```diff
@@ -1,23 +1,39 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from django.db.models import QuerySet, F, Q
+from db.models import Customer, LoyaltyProgram, LoyaltyProgramParticipant
 
 
 def all_loyalty_program_names() -> QuerySet:
-    pass
+    return LoyaltyProgram.objects.values_list(
+        "name",
+        "bonus_percentage")
```

Comment:
I'm not sure about the parentheses here.
it should be either
`values(...)`
or
```
values(
    ...
)
```